app/Models/Payment.php

```
@@ -331,10 +331,12 @@
     public function getOngoingPayment($request = array())
     {
-        return Payment::where([
-            'loan_account_id' => $request['loan_account_id'],
-            'status' => 'open'
-        ])->get()->first();
+        return DB::transaction(function () use ($request) {
+            return Payment::where([
+                'loan_account_id' => $request['loan_account_id'], 
+                'status' => 'open'
+            ])->lockForUpdate()->first();
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'getOngoingPayment'. (Clean Code Rules)

(StaticAccess)